VersionInfo.cs

```diff
@@ -1,9 +1,9 @@
 using System.Reflection;
 
 // This should be the same version as below
-[assembly: AssemblyVersion("3.0.0")]
-[assembly: AssemblyFileVersion("3.0.0")]
-[assembly: AssemblyInformationalVersion("3.0.0")]
+[assembly: AssemblyVersion("3.1.0")]
+[assembly: AssemblyFileVersion("3.1.0")]
```

Comment:
One of these (this one if I recall correctly) should only change for major versions